### PR TITLE
Filters

### PR DESCRIPTION
**Pull request by itsmebipul00**
Files not to check : backend
Large amount of lines of code is because of prettier which wasnt working last time
Files to be checked: actionProvides, reducers, Login, Register, Wishlist and Product Listing Screens

### CODE REVIEW

**Review by sanu29 — APPROVED** [review]
Great work. Naming conventions Good. ES6 used everywhere. Components made where needed. 
All good to procced.

File: src/App.css [hunk 1]
```diff
@@ -1,59 +1,81 @@
...
-  --fs-200: 0.5rem;
-
-  --ff-serif-ibm: "IBM Plex Sans", sans-serif;
+	--fs-850: 3rem;
+	--fs-800: 2rem;
+	--fs-700: 1.75rem;
+	--fs-600: 1.5rem;
+	--fs-500: 1.25rem;
+	--fs-400: 1rem;
+	--fs-300: 0.75rem;
```
Comment (developer-anand) [inline src/App.css:14]: Just a suggestion if possible use px here for small values.

File: src/Components/Filters/Filters.css [hunk 2]
```diff
@@ -1,17 +1,109 @@
...
+@media only screen and (min-width: 40em) {
+	.price-range {
+		width: 30%;
+		display: list-item;
+	}
+	.react-icon-buttons {
+		margin-right: 1rem;
+	}
+}
+.sort-by-heading {
```
Comment (developer-anand) [inline src/Components/Filters/Filters.css:83]: use px if you can.

**Review by developer-anand — APPROVED** [review]
Clean code, Good naming convention.
Can be merged.

**Pull request merged**